pack.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
   - salt
   - cfg management
   - configuration management
-version: 1.0.1
+version: 2.0.1
```

Comment:
The changelog was referring to `2.0.0`, while we actually released `2.0.1` here :man_shrugging: